tcl/response.tcl

```diff
@@ -383,6 +400,9 @@ proc qc::response2html_snippet {} {
                 status {
                     set status $value
                 }
+                data {
+                    set data_html [qc::response_tson2html_fragment $value]
```

Comment:
`response2html` needs to handle the `data` property somehow (even if just by ignoring it), I went with including a `<div>` with class `data`, containing:

1. Each property of an object as a div, with an id for the property name and the div contents for the value
2. Each element of an array as a div with class "array-element", the contents being the element value
3. Anything explicitly declared as a string, number, or boolean, wrapped in a span with a class to indicate the data type
4. Anything else included directly, as escaped html.